airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -119,6 +150,17 @@ def __init__(
         except ValueError:
             logger.setLevel(logging.WARNING)
 
+    def _get_field(self, extra_dict, field_name):
+        prefix = 'extra__wasb__'
+        if field_name.startswith('extra_'):
+            raise ValueError(
+                f"Got prefixed name {field_name}; please remove the '{prefix}' prefix "
+                f"when using this method."
+            )
+        if field_name in extra_dict:
+            return extra_dict[field_name] or None
```

Comment:
Is returning `None` needed to be returned when the key is present in the dict? WDYT?

Reply:
i like to treat empty string as none in this context because UI creates all custom fields with empty string even if not supplied